rust/xaynet-client/src/mobile_client/participant/mod.rs

```diff
@@ -32,19 +34,93 @@ pub struct ParticipantState {
     pub keys: SigningKeyPair,
     // Mask config
     pub aggregation_config: AggregationConfig,
+    pub max_message_size: MaxMessageSize,
 }
 
 #[derive(Serialize, Deserialize)]
 pub struct ParticipantSettings {
     pub secret_key: ParticipantSecretKey,
     pub aggregation_config: AggregationConfig,
+    pub max_message_size: MaxMessageSize,
+}
+
+const MESSAGE_HEADER_LEN: usize = 136;
+const ENCRYPTION_HEADER_LEN: usize = 48;
+pub const MIN_MESSAGE_SIZE: usize = MESSAGE_HEADER_LEN + ENCRYPTION_HEADER_LEN + 1;
```

Comment:
would be good to have some documentation here, especially regarding the `+1`, i guess it means that the payload must be at least one byte?